Source/WebCore/platform/network/DNS.cpp

```diff
@@ -76,4 +76,50 @@ std::optional<IPAddress> IPAddress::fromString(const String& string)
     return std::nullopt;
 }
 
+IPAddress IPAddress::isolatedCopy() const
+{
+    if (isIPv4())
+        return IPAddress { ipv4Address() };
+
+    if (isIPv6())
+        return IPAddress { ipv6Address() };
+
+    ASSERT_NOT_REACHED();
```

Comment:
Could we have a release assert here? I always prefer those.

Reply:
Sounds good — changed to `RELEASE_ASSERT_NOT_REACHED();`.